docs/man_pages/project/configuration/edit-configuration.md

```diff
@@ -34,4 +34,6 @@ Command | Description
 [prop add](prop-add.html) | Enables more options for the selected project property, if the property accepts multiple values.
 [prop remove](prop-remove.html) | Disables options for the selected project property, if the property accepts multiple values.
 [prop set](prop-set.html) | Sets the selected project property and overwrites its current value.
+[web-view](web-view.html) | Lists all supported web views for different mobile platforms.
+[web-view set](web-view-set.html) | Sets the selected web view for the current project and updates the integrated plugins to match it.
```

Comment:
Sets the selected web view for the current project.

(The information about the plugin updates will be inside the extended help.)